deny.toml

```diff
@@ -16,10 +16,6 @@ notice = "warn"
 [bans]
 multiple-versions = "allow"
 
-[[bans.deny]]
```

Comment:
This should still be banned, and maybe upstreamed to operator-templating.

Reply:
The `time` crate is not used. Does it make sense to ban unused crates? If yes, why exactly `time`?

Reply:
operator-rs depends on `k8s-openapi`, which depends on `chrono`, which depends on `time` 0.1 unless everyone who depends on it is religious about using `default-features = false` (and nobody explicitly enables the feature `oldtime`). This has happened before, and we'd like to avoid it happening again in the future.

`time` 0.1 is unmaintained, and has a bunch of outstanding vulnerability advisories (such as https://rustsec.org/advisories/RUSTSEC-2020-0071.html), which cause false positive errors elsewhere even if we never use them ourselves.

> Does it make sense to ban unused crates?

Yes, bans exist specifically to avoid specific crates getting pulled in as transitive dependencies.